blocks/library/latest-posts/index.js

```diff
@@ -38,25 +49,83 @@ registerBlockType( 'core/latestposts', {
 		}
 
 		render() {
-			const { latestPosts } = this.state;
-
-			if ( ! latestPosts.length ) {
-				return (
-					<Placeholder
-						icon="update"
-						label={ __( 'Loading latest posts, please wait' ) }
-					>
-					</Placeholder>
-				);
-			}
+			const { latestPosts, focus } = this.state;
+
+			const alignments = [
+				{
+					icon: 'align-left',
+					title: wp.i18n.__( 'Align left' ),
+					isActive: ( { align } ) => 'left' === align,
+					value: 'left',
+				},
+				{
+					icon: 'align-center',
+					title: wp.i18n.__( 'Align center' ),
+					isActive: ( { align } ) => ! align || 'center' === align,
+					value: 'center',
+				},
+				{
+					icon: 'align-right',
+					title: wp.i18n.__( 'Align right' ),
+					isActive: ( { align } ) => 'right' === align,
+					value: 'right',
+				},
+				{
+					icon: 'align-wide',
+					title: __( 'Wide width' ),
+					isActive: ( { align } ) => 'wide' === align,
+					value: 'wide',
+				},
+				{
+					icon: 'align-full-width',
+					title: __( 'Full width' ),
+					isActive: ( { align } ) => 'full' === align,
+					value: 'full',
+				},
+			];
 
 			return (
-				<div className="blocks-latest-posts">
-					<ul>
-						{ latestPosts.map( ( post, i ) =>
-							<li key={ i }><a href={ post.link }>{ post.title.rendered }</a></li>
-						) }
-					</ul>
+				<div>
+					{ 0 === latestPosts.length ?
+						<Placeholder
+							icon="update"
+							label={ __( 'Loading latest posts, please wait' ) }
+						>
+						</Placeholder>
+						:
+						<div className="blocks-latest-posts">
+							<ul>
+								{ latestPosts.map( ( post, i ) =>
+									<li key={ i }><a href={ post.link }>{ post.title.rendered }</a></li>
+								) }
+							</ul>
+						</div>
+					}
+					{ /* focus && */
```

Comment:
I suspect that it's because you're saving the focus in the state from the initial props but when the prop is being updated the state is not being updated.

I suggest you avoid storing the focus in the local state and use the props directly.